src/hotspot/share/runtime/os.inline.hpp

```diff
@@ -27,6 +27,37 @@
 
 #include "runtime/os.hpp"
 
+#include OS_HEADER(os)
 #include OS_HEADER_INLINE(os)
+#include OS_CPU_HEADER_INLINE(os)
+
+// Below are inline functions that are rarely implemented by the platforms.
+// Provide default empty implementation.
+
+#ifndef PLATFORM_PRINT_NATIVE_STACK
```

Comment:
Could you rename this to HAVE_PLATFORM_PRINT_NATIVE_STACK?

Reply:
Fixed